lib/galaxy/managers/workflows.py

```diff
@@ -320,6 +323,31 @@ def attach_stored_workflow(self, trans, workflow):
                 trans.sa_session.commit()
             return stored_workflow
 
+    def get_workflow_by_trs_id_and_version(self, sa_session, trs_id: str, trs_version: str, user_id: int) -> Optional[model.Workflow]:
+        def to_json(column, keys: List[str]):
```

Comment:
Is there a reason for passing a list? I see only 2 calls, both pass a list of 1, and then you iterate over the items in the list (and using only the last item anyway). Why not just pass the one key?

Reply:
I wanted to generalize the function eventually, but this can actually already be a string or a list, so this isn't necessary in any case.